File: API/database/cliente.py
from sqlalchemy import Column, Integer, String, Boolean, DateTime, Text
from sqlalchemy.sql import func
from database.db import Base


class Cliente(Base):
    """Clientes B2B (módulo Ventas Flask) y referencia para pedidos."""

    __tablename__ = "clientes"

    id = Column(Integer, primary_key=True, index=True)
    nombre = Column(String(200), nullable=False)
    email = Column(String(120), nullable=False, unique=True, index=True)
    telefono = Column(String(30), nullable=True)
    activo = Column(Boolean, default=True)
    notas = Column(Text, nullable=True)
    fecha_registro = Column(DateTime(timezone=True), server_default=func.now())

File: Laravel/app/Models/Macuin/Cliente.php
<?php

namespace App\Models\Macuin;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;

class Cliente extends Model
{
    protected $table = 'clientes';

    public $timestamps = false;

    protected $casts = [
        'activo' => 'boolean',
        'fecha_registro' => 'datetime',
    ];

    protected $fillable = [
        'nombre',
        'email',
        'telefono',
        'activo',
        'notas',
    ];

    public function pedidos(): HasMany
    {
        return $this->hasMany(Pedido::class, 'cliente_id');
    }
}
